drf_extra_fields/relations.py

```diff
@@ -43,6 +43,11 @@ def to_representation(self, data):
             data, context=self.context, **self.presentation_serializer_kwargs
         ).data
 
+    def bind(self, field_name, parent):
+        if self.presentation_serializer == "self":
+            self.presentation_serializer = parent.__class__
```

Comment:
I am assuming we can't implement this because we can't access to parent class when many=True.

How about instead of self we ask for a serializer path.  For example: "tests.test_relations.RecursiveSerializer"

This can be imported via django's built in method `import_string`

Since at the time of binding or init this class is not fully loaded import_string will not find the self referenced serializers. Because of that this logic should be moved to to_representation.

This can also be used to reference other serializers other than "self" to solve circular import problems.

What do you think @cobang @KaratasFurkan @joshcorne ?

I've tried this and it's working as expected.

Reply:
Thank you. It makes sense to me. I couldn't find a way to work `self` and `many=True` together.

Reply:
@KaratasFurkan Do you have time to make the necessary changes? If not please let me know i will continue and complete the PR.

Reply:
I will do it today, thank you.

Reply:
@KaratasFurkan Thank you. Could you please also update the documentation?

Reply:
Sure 👍